kong/plugins/grpc-gateway/deco.lua

```diff
@@ -169,21 +169,42 @@ function deco.new(method, path, protofile)
   }, deco)
 end
 
+local function get_field_type(typ, field)
+  local _, _, field_typ = pb.field(typ, field)
+  return field_typ
+end
+
+local function encode_fix(v, typ)
+  if typ == "bool" then
+    -- special case for URI parameters
+    return v and v ~= "0" and v ~= "false"
```

Comment:
```suggestion
    return v and v ~= "0" and v ~= "false" and v ~= "null"
```

Should we handle the string `"null"`?

Reply:
We should not.
Here's the official implementation's response to null:
```json
{
	"code": 3,
	"message": "parsing field \"boolean\": strconv.ParseBool: parsing \"null\": invalid syntax",
	"details": []
}
```